components/notebook-controller/controllers/culling_controller.go

```diff
@@ -381,6 +388,16 @@ func StopAnnotationIsSet(meta metav1.ObjectMeta) bool {
 	return false
 }
 
+func DisableCullingAnnotationIsSet(meta metav1.ObjectMeta) bool {
```

Comment:
small nit, shouldn't this function start from lowercase since it's not meant to be used from outside this go module?

Reply:
I was following the style of the current code using `StopAnnotationIsSet` above. Agreed it should be lowercase